### PR TITLE
chore: Replace Background Image for smaller size

### PR DESCRIPTION
**Pull request by JeremyParish69**
## What is the purpose of the change

Update the Frontend background image to smaller size to improve user load times

## Brief Changelog

- update pages/index background image reference
- replace Margined^2.png with Margined-frontend.png, which is much smaller

## Testing and Verifying

This change has been tested locally by rebuilding the website and verified content and links are expected

Validation passes

### CODE REVIEW

**Comment by jonator** [comment]
I think we should make it a little bigger. It doesn't stick to the bottom edge of the page. Or move increase the y coordinate a bit to keep it at the bottom.

Screenshot 2023-11-10 at 3 59 41 PM

**Review by MaxMillington — APPROVED** [review]

**Comment by JeremyParish69** [comment]
> I think we should make it a little bigger. It doesn't stick to the bottom edge of the page. Or move increase the y coordinate a bit to keep it at the bottom.
> 
> Screenshot 2023-11-10 at 3 59 41 PM

Any recommended settings? Not sure how to pin it to the bottom...

**Comment by JeremyParish69** [comment]
I've increased the size. Is this better from your view, @jonator ?

File: packages/web/pages/index.tsx [hunk 1]
```diff
@@ -57,12 +57,12 @@ const Home = ({ ads }: InferGetStaticPropsType<typeof getStaticProps>) => {
         >
           <g>
             <ProgressiveSvgImage
-              lowResXlinkHref="/images/Margined^2-low.png"
-              xlinkHref="/images/Margined^2.png"
+              lowResXlinkHref="/images/Margined-frontend.png"
+              xlinkHref="/images/Margined-frontend.png"
```
Comment (ValarDragon) [inline packages/web/pages/index.tsx:61]: ah awesome this also fixes the issue of my laptop downloading both

**Review by jonator — APPROVED** [review]

**Pull request merged**